app/vmui/packages/vmui/src/components/Chart/Line/Legend/Legend.tsx

```diff
@@ -7,10 +7,11 @@ import "./style.scss";
 interface LegendProps {
   labels: LegendItemType[];
   query: string[];
+  anomalyView?: boolean;
```

Comment:
nit: i think it is better to use boolean naming, like isAnomalyView or something similar